polaris-react/src/components/Filters/Filters.tsx

```diff
@@ -414,7 +393,7 @@ export function Filters({
           size="micro"
           onClick={handleClearAllFilters}
           removeUnderline
-          variant="tertiary"
+          variant="monochromePlain"
```

Comment:
[Previously](https://github.com/Shopify/polaris/commit/4caed28a77c7c5b4cf78dc8071631e7054173740#diff-f21b98ea8a1f295706a4342f1e9ce543562f6c47d6d0682367e90bb7c23961d8) this was `monoChrome=true` and `plain=true`, the equivalent variant value should be `monochromePlain` not `tertiary`.